@ndlib/gatsby-theme-marble/src/components/Internal/Seo/SeoContent/index.js

```diff
@@ -1,67 +1,49 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import Helmet from 'react-helmet'
-import CanonicalLink from './CanonicalLink'
-// import SchemaLink from './SchemaLink'
-import MetaTagGroup from './MetaTagGroup'
-import { getOpenGraph, getTwitter } from './data'
+import GatsbySeoNext from './GatsbySeoNext'
 
 export const SeoContent = ({
   title,
   url,
   description,
   image,
-  lang,
   pathname,
   author,
   siteTitle,
   siteUrl,
+  lang,
   noIndex,
+  noFollow,
 }) => {
-  const openGraph = getOpenGraph(url, title, description, image)
-  const twitter = getTwitter(author, title, description, image)
-  const titleFix = title.includes('Mirador Viewer') ? title : `${title} | ${siteTitle}`
-  let indexable = null
-  if (noIndex === true) {
-    indexable = (
-      <Helmet>
-        <meta name='robots' content='noindex' />
-      </Helmet>
-    )
-  }
   return (
-    <React.Fragment>
-      <Helmet
-        htmlAttributes={{ lang }}
-        title={title}
-        titleTemplate={title === siteTitle ? `${siteTitle}` : `${titleFix}`}
-        meta={[
-          {
-            name: `description`,
-            content: description,
-          },
-        ]}
-      />
-      <CanonicalLink base={siteUrl} pathname={pathname} />
-      <MetaTagGroup tags={openGraph} />
-      <MetaTagGroup tags={twitter} />
-      {indexable}
-    </React.Fragment>
+    <GatsbySeoNext
+      title={title}
+      description={description}
+      author={author}
+      url={url}
+      image={image}
+      pathname={pathname}
+      siteTitle={siteTitle}
+      base={siteUrl}
+      lang={lang}
+      noIndex={noIndex}
+      noFollow={noFollow}
+    />
   )
 }
```

Comment:
Since the only thing SeoContent is doing now is rendering your GatsbySeoNext component, can the content of GatsbySeoNext just be put directly in SeoContent?

Reply:
It certainly can. I left it because I had considered a potential JSON-LD component to be downstream from SeoContent, but if we are going use a transformer instead of a component than I'll just move it up.

Reply:
I moved the code for GatsbySeoNext into SeoContent and updated the unit tests. In order for this to function the shadow component needs to be removed from marble-web. I have a PR for that [here](https://github.com/ndlib/marble-web/pull/24).